Python/Product/PythonTools/PythonTools/Project/ImportWizard/ImportSettings.cs

```diff
@@ -204,6 +204,8 @@ internal class ImportSettings : DependencyObject {
                 _isAutoGeneratedProjectPath = true;
             }
 
+            var startup = this.StartupFile;
```

Comment:
Could you add a comment here explaining the need for a workaround? Otherwise this might get accidentally reverted in some future refactoring.

Reply:
Sure, just added.